crates/uv/tests/cache_clean.rs

```diff
@@ -25,15 +25,20 @@ fn clean_all() -> Result<()> {
         .assert()
         .success();
 
-    uv_snapshot!(context.filters(), context.clean().arg("--verbose"), @r###"
+    let mut filters = context.filters();
+    filters.extend([
+        ("Removed 28 files", "Removed [N] files"),
+        ("Removed 27 files", "Removed [N] files"),
```

Comment:
Can you try using `context.with_filtered_counts()`?

Reply:
Ah yeah nice tip. I had thought about doing that same thing (re-implementing it I mean, since I didn't know about that routine) but thought that it might be taking too much away from what we're trying to test here. That is, if this test said, "Removed 0 files," then we'd probably want that to result in a failure.